src/client/pages/OfferNew/Tabs/Tab.tsx

```diff
@@ -12,9 +12,11 @@ const TabContainer = styled.button<{ selected?: boolean }>`
   display: flex;
   font-size: 1.25rem;
   line-height: 1.4;
-  padding: 0 1rem 0.25rem 0;
+  padding: 0 1rem calc(0.25rem + 2px) 0;
```

Comment:
So is this due to the `button` margin thingy in Safari? Or why is `2px` added to the padding? If this is what we have to do I think a comment might be good to explain why we have this 😇

Reply:
Oh that's just a coincidence but I can see the confusion.

These 2px is a reference to the height of the underline component. Perhaps we can break that out into a named constant.

Reply:
That would be great! 👍